ssc/cmod_fresnel_physical_iph.cpp

```diff
@@ -153,7 +153,17 @@ static var_info _cm_vtab_fresnel_physical_iph[] = {
 
     // Heat Sink
 
-    {SSC_INPUT,     SSC_NUMBER,         "pb_pump_coef",                "Pumping power to move 1kg of HTF through PB loop",                                      "kW/kg",               "",                              "Heat Sink",           "*",                "",                 ""},
+    { SSC_INPUT,    SSC_NUMBER,         "pb_pump_coef",                "Pumping power to move 1kg of HTF through PB loop",                                      "kW/kg",               "",                             "Heat Sink",            "*",                "",                 "" },
+
+    { SSC_INPUT,    SSC_NUMBER,         "hs_type",                     "0: ideal model, 1: physical steam model",                                               "",                    "",                             "Heat Sink",            "?=0",              "",                 "" },
```

Comment:
This value defaults to zero and cannot be set in the UI currently - correct?

Reply:
Correct, hs_type defaults to zero and cannot be set via UA